packages/query/src/graphql/graphql.module.ts

```diff
@@ -41,8 +42,8 @@ const SCHEMA_RETRY_NUMBER = 5;
   providers: [ProjectService],
 })
 export class GraphqlModule implements OnModuleInit, OnModuleDestroy {
-  private apolloServer: ApolloServer;
-  private dbType: SUPPORT_DB;
+  private apolloServer!: ApolloServer;
+  private dbType!: SUPPORT_DB;
```

Comment:
Rather than adding `!` can you please use private getters/setters that have an assertion.

Here's an example https://github.com/subquery/subql/blob/main/packages/node-core/src/indexer/store.service.ts#L76-L78